src/Models/AlbumsModel.php

```diff
@@ -40,5 +40,26 @@ public function getAlbumSongCount(int $albumId): Album
         $query->execute(['albumId' => $albumId]);
         return $query->fetch();
     }
+
+    public function getPopularAlbums(): array
```

Comment:
Can you add a comment above this, exactly the same as the one above getArtistAlbums